build.gradle

```diff
@@ -14,20 +14,20 @@ allprojects {
     }
 }
 
-// We compile the library using Java 1.6 compatibility
+// We compile the library using Java 1.7 compatibility
 // in order to ensure interoperability with older Android platforms.
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
 
 // load version number from file
-def config = new ConfigSlurper().parse(new File("${projectDir}/src/main/resources/version.properties").toURI().toURL())
+def config = new ConfigSlurper().parse(new File("${projectDir}/src/main/resources/tusVersion/version.properties").toURI().toURL())
```

Comment:
Can we include the full package name is the path to the version file? e.g.

> resources/io/tus/java/client/version.properties

This way we avoid a name conflict if we also want to have a `version.properties` file in tus-android-client.

Reply:
of course :)